charts/speedtest-prometheus/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: speedtest-prometheus
 description: Prometheus Exporter for the official Speedtest CLI
 type: application
-version: 2.0.0
+version: 2.0.1
```

Comment:
I would bump to 2.1.0, since this is a new feature :) thanks!

Reply:
Fair point! Done, thanks!